packages/node/src/ethereum/api.ethereum.ts

```diff
@@ -279,7 +280,10 @@ export class EthereumApi implements ApiWrapper {
     return this.client.getBlock(heightOrHash);
   }
 
-  private async getBlockPromise(num: number, includeTx = true): Promise<any> {
+  private async getBlockPromise(
+    num: number,
+    includeTx = true,
+  ): Promise<IBlock<any>> {
```

Comment:
I don't think we need to wrap this in IBlock here, this method is private, we can do it on the return in `fetchBlock`/`fetchLightBlock`